### PR TITLE
Fix/delete cache item

### PR DESCRIPTION
**Pull request by DavidCramer**

### CODE REVIEW

File: php/cache/class-cache-point.php [hunk 1]
```diff
@@ -476,6 +476,48 @@ protected function get_parent_cache_point( $url ) {
...
+		if ( ! is_int( $cache_point_id_url ) ) {
+			$cache_point = $this->get_cache_point( $cache_point_id_url );
+		} else {
+			$cache_point = get_post( $cache_point_id_url );
+		}
+		if ( ! is_null( $cache_point ) ) {
+
+			$params = array(
+				'post_type'              => self::POST_TYPE_SLUG,
+				'posts_per_page'         => 1,
```
Comment (pereirinha) [inline php/cache/class-cache-point.php:498]: @DavidCramer 

Any reason why we don't do in batches of 100, for instance?

**Review by pereirinha — COMMENTED** [review]
@DavidCramer 

Left you a minor comment.

File: php/cache/class-cache-point.php (hunk 1 above)
Comment (DavidCramer) [reply php/cache/class-cache-point.php:500]: Typo! fixing.

**Review by pereirinha — APPROVED** [review]
Works great @DavidCramer 
Thanks

**Pull request merged**